qwik-graphql-tailwind/src/components/icons/merged-pr.icon.tsx

```diff
@@ -0,0 +1,15 @@
+import { component$, useStylesScoped$ } from '@builder.io/qwik';
+import { iconStyles } from './icons.styles';
+import { IconProps } from './types';
+
+export const MergedPrIcon = component$(({ className }: IconProps) => {
```

Comment:
mergedPricon is just PullrequestIcon with color of purple and we already have pullrequest icon